### PR TITLE
Improve recurring Ansible jobs for host and host group

### PR DESCRIPTION
**Pull request by Lennonka**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3

### CODE REVIEW

File: guides/common/modules/proc_scheduling-a-recurring-ansible-job.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+. Define the repetition frequency, start time, and date of the first run in the *Create New Recurring Ansible Run* window.
+. Click *Submit*.
+. Optional: View the scheduled Ansible job in host overview or by navigating to *Ansible* > *Jobs*.
+
+.Scheduling a Recurring Job for a Host Group
+. In the {ProjectWebUI}, navigate to *Configure* > *Host groups*.
+. In the *Actions* column, select *Configure Ansible Job* for the host group you want to schedule an Ansible roles run for.
+. Click *Schedule recurring job*.
+. Define the repetition frequency, start time, and date of the first run in the *Create New Recurring Ansible Run* window.
+. Click *Submit*.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_scheduling-a-recurring-ansible-job.adoc:19]: IMHO this is idealy a separate procedure. Split move to a separate file.

File: guides/common/modules/proc_scheduling-a-recurring-ansible-job.adoc [hunk 2]
```diff
@@ -0,0 +1,19 @@
+[id="scheduling-a-recurring-ansible-job_{context}"]
+= Scheduling a Recurring Ansible Job
+
+You can schedule recurring jobs to run Ansible roles on hosts or host groups.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_scheduling-a-recurring-ansible-job.adoc:4]: Adapt accordingly.

**Review by maximiliankolb — CHANGES_REQUESTED** [review]
the rest LGTM!

**Comment by Lennonka** [comment]
Done.

**Review by maximiliankolb — APPROVED** [review]
LGTM!

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: ✅ 
   1fbe5aeaa..96096f3b2  3.3 -> 3.3
   619bc8c69..f9b66769e  3.2 -> 3.2
   c7c3a0ab3..82627a516  3.1 -> 3.1

**Pull request merged**